src/main/java/cat/udl/eps/softarch/linkapp/exception/ForbiddenException.java

```diff
@@ -0,0 +1,7 @@
+package cat.udl.eps.softarch.linkapp.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value= HttpStatus.FORBIDDEN, reason = "Unauthorized")
+public class ForbiddenException extends RuntimeException {}
```

Comment:
There is a built-in from spring that I think it does the same function. We have used it in [this line](https://github.com/UdL-EPS-SoftArch/LinkApp-API/blob/main/src/main/java/cat/udl/eps/softarch/linkapp/handler/MeetEventHandler.java#L41)